### PR TITLE
Feature/m brittany w8ben

### PR DESCRIPTION
**Pull request by brimagnin**

### CODE REVIEW

File: content/api-references/accounts/accounts.md [hunk 1]
```diff
@@ -550,6 +557,79 @@ _Some server error occurred. Please contact Alpaca._
...
+    "permanent_address_country": "Canada",
+    "permanent_address_street": "50 Main St",
+    "reference_number": "abc123",
+    "residency": "Canada",
+    "revision": "7-2017",
+    "tax_id_ssn": "123-00-456",
+    "timestamp": "2021-06-14T09:31:05Z",
+    "income_type": "interest",
+    "signer_full_name": "Mr. Signing User" },
+    "mime_type": "application/json"
```
Comment (jerwheaton) [inline content/api-references/accounts/accounts.md:599]: Don't need this field with `content_data`

File: content/api-references/accounts/accounts.md [hunk 2]
```diff
@@ -550,6 +557,79 @@ _Some server error occurred. Please contact Alpaca._
...
+Note: The dates collected on the form are in a slightly different format than how they need to be submitted via Accounts API. It is requested by the user on the form in MM-DD-YYYY, but should be submitted as YYYY-MM-DD.
+
+#### Sample Request Body
+
+```json
+{
+"document_type": "w8ben",
+"content_data" : {
+    "additional_conditions": "None",
+    "capacity_acting": "broker",
```
Comment (jerwheaton) [inline content/api-references/accounts/accounts.md:577]: Use `self` here, I think that'll be the most common value
```suggestion
    "capacity_acting": "SELF",
```

**Review by jerwheaton — APPROVED** [review]

File: content/api-references/accounts/accounts.md [hunk 3]
```diff
@@ -77,7 +77,13 @@ The Accounts API allows you to create and manage the accounts under your brokera
       "document_sub_type": "passport",
       "content": "QWxwYWNhcyBjYW5ub3QgbGl2ZSBhbG9uZS4=",
       "mime_type": "image/jpeg"
+    }, 
+    {
+      "document_type": "w8ben",
+      "content": "RFerjiUxFUTXLQ7pMEckpWYjLPviLRZdrc4=",
+      "mime_type": "application/pdf"
     }
+
```
Comment (umitanuki) [inline content/api-references/accounts/accounts.md:86]: Do you need an extra line here?

File: content/api-references/accounts/accounts.md [hunk 4]
```diff
@@ -550,6 +557,78 @@ _Some server error occurred. Please contact Alpaca._
 
 ---
 
+## **International Accounts**
+
+#### W-8 BEN
+
+For certain individuals, a W-8 BEN form should be submitted at onboarding. If the individual is not a registered U.S. taxpayer (not subject to a W-9), the W-8 BEN form may need to be submitted. The IRS explains [here](https://www.irs.gov/instructions/iw8ben) which individuals this applies to and provides instructions on completing the form. Every three years, in addition to the calendar year it was signed, a new W-8 BEN form must be submitted.
```
Comment (umitanuki) [inline content/api-references/accounts/accounts.md:564]: Is `W-8BEN` more correct than `W-8 BEN`?

**Pull request merged**